swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id, condition)
+
+        self.fabric = fabric
+        self.category = "Clothing"
```

Comment:
Again, with the modifications mentioned in `item.py`, we could be able to remove this attribute and pull it from the class name with `self.__class__.__name__`